src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -266,11 +269,13 @@ private static bool GetTls13Support()
                 // [ActiveIssue("https://github.com/dotnet/runtime/issues/1979")]
                 return false;
             }
-            else
+            else if (IsLinux)
```

Comment:
Should this be `else if (IsOpenSslSupported)` ?

Reply:
updated. I added IsOpenSslSupported in second pass and missed this place.